### DISCUSSION TITLE
Slider accuracy and pp

### DISCUSSION THREAD

**Discussion by Flamiii** [ppy/osu#21096] [Ideas] [answered]
I like that having slider accuracy on is the new default in osu!lazer, but once osu!lazer has leaderboard/pp submission like stable does, I'm not really sure if there's much of a reason to ever keep slider accuracy on.

In the classic mod settings, there are two main options regarding this setting:
1. Keep slider accuracy on, missing a sliderend doesn't impact your accuracy as much and sliderends don't count towards combo (osu!lazer default)
2. Turn slider accuracy off, missing a sliderend gives you a 100 and sliderends count towards combo (stable default)

The problem I currently see with this is that the second option is almost universally easier on the vast majority of maps in the game, especially when it comes to the generic farm maps that players often play for PP. These maps usually won't have many tricky sliders at all, so turning slider accuracy off is a no-brainer. The only time that the first option would ever be more desirable is when someone is playing a tech or gimmick map that has a ton of difficult sliders, but these maps generally give very little pp regardless (with a few exceptions). This is unfortunate because slider accuracy is really nice to have in the game, but it feels like there's almost no reason to play with it on, as on most maps, it will result in you having lower accuracy than you would otherwise, resulting in less pp.

I believe that the current pp system doesn't award accuracy pp for hitting sliders, which makes sense since slider accuracy isn't a thing in stable. In osu!lazer, the pp system treats sliders in the same way as stable regardless of whether you have slider accuracy on or off. I think that it would make more sense to award accuracy pp for hitting sliders if you have slider accuracy on, and as a result, maps should give a bit more pp if you have slider accuracy on. This would make it so that there's a bit more of an incentive to play with slider accuracy, as if you know you can get good accuracy on a map, you can potentially get more pp.

**Comment by Rekunan** [comment]
Fwiw, there are future plans to bypass the need to look at the score's accuracy and just use the UR for accuracy pp, once it is added to the score data in Lazer. For the case of stable plays (which won't be able to give UR), there's a WIP rework that estimates UR: #15091

**Reply by apollo-dw** [reply]
> Fwiw, there are future plans to bypass the need to look at the score's accuracy and just use the UR for accuracy pp

For the record, this is not true currently.

**Comment by apollo-dw** [comment, marked as answer]
100%, lazer scores (without classic mod) will definitely have sliders included in acc pp.
